f-type/cpu.h

```diff
@@ -31,27 +31,27 @@ typedef enum {
     AM_INDIRECT_X,
     AM_INDIRECT_Y,
     AM_RELATIVE
-} addressing_mode;
+} AddressingMode;
 
 
 typedef union {
     uint16_t addr;
     uint8_t immediate_value;
     int8_t relative_addr;
-} op_param;
+} OpParam;
 
-typedef struct cpu_state cpu_state;
-typedef struct opcode opcode;
-struct opcode {
+typedef struct CpuState CpuState;
+typedef struct OpCode OpCode;
+struct OpCode {
```

Comment:
Opcode does not have a uppercase C, it's a single word
https://en.wikipedia.org/wiki/Opcode

Reply:
Sounds good.